### PR TITLE
Added EBSCONet workflow script

### PR DESCRIPTION
**Pull request by Roman-Fedynyshyn**

### CODE REVIEW

**Comment by ArtVronik** [comment]
for commit be3c91c12adf2847970ca069d5ae08f9c9e46c24:
1. 'CSV Data Set Config - credentials' could be as a parameter 
2. HTTP Header Manager - could be like:
	1 - Plan level with:
		Accept
		x-okapi-tenant
		content-type
	2 - 2nd thread
		x-okapi-token

3. API_KEY, EDGEHOSTNAME are never used
4. HTTP Request Defaults should be at plan level to avoid duplication
5. EBSCONET Okapi is infinite but the duration isn't specified
	EBSCONET Okapi > 1st HTTP Header Manager - tenant is hardcoded
	put  /ebsconet/orders/order-lines/{polineNumber} could be parametrized via high-level jsr sampler
6. edgeebsconet Edge - is toggled off so unused at all				
	put /orders/order-lines/{polineNumber}?type=EBSCONET&apiKey={API_KEY} could be parametrized via high-level jsr sampler (more over parameters are the same)

File: workflows-scripts/EBSCONet/EBSCONET.jmx [hunk 1]
```diff
@@ -0,0 +1,413 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.4.1">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="EBSCONET" enabled="true">
```
Comment (mtraneis) [inline workflows-scripts/EBSCONet/EBSCONET.jmx:4]: Please change the test name and anywhere else that refers to "EBSCONET" to "EBSCONet-order-renewal" (or some variant of the casing that adheres to the PTF established naming convention). 

```suggestion
    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="EBSCONet Order Renewal" enabled="true">
```

File: workflows-scripts/EBSCONet/EBSCONET.jmx [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+          <stringProp name="shareMode">shareMode.all</stringProp>
+          <boolProp name="stopThread">true</boolProp>
+          <stringProp name="variableNames">polineNumber</stringProp>
+        </CSVDataSet>
+        <hashTree/>
+        <ConfigTestElement guiclass="HttpDefaultsGui" testclass="ConfigTestElement" testname="HTTP Request Defaults" enabled="true">
+          <elementProp name="HTTPsampler.Arguments" elementType="Arguments" guiclass="HTTPArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+            <collectionProp name="Arguments.arguments"/>
+          </elementProp>
+          <stringProp name="HTTPSampler.domain">${EDGEHOSTNAME}</stringProp>
```
Comment (mtraneis) [inline workflows-scripts/EBSCONet/EBSCONET.jmx:328]: If the edge thread group was used, how can the ${EDGEHOSTNAME} parameter be passed in from carrier-io Jenkins job (using which text field)? 

Same question for the ${API} variable.

Please include this information in the README file.

**Comment by mtraneis** [comment]
> 3\. API_KEY, EDGEHOSTNAME are never used

These parameters would be used if the edge-threadgroup is enabled. 

How is it to implement Artem's suggestion here: "put /orders/order-lines/{polineNumber}?type=EBSCONET&apiKey={API_KEY} could be parametrized via high-level jsr sampler (more over parameters are the same)"?

**Pull request merged**